WGLC feedback. (#97)
network-quality/draft-ietf-ippm-responsiveness: draft-ietf-ippm-responsiveness.md

Over that period of time, a large number of samples will have been collected.
These may be affected by noise in the measurements, and outliers. Thus, to aggregate these

> we suggest using a single sided trimmed mean at the `TMP` (Trimmed Mean Percentage - default to 95%) percentile, thus providing the following numbers:

Reviewer: ```suggestion
we suggest using a single-sided trimmed mean at the `TMP` (Trimmed Mean Percentage - default to 95%) percentile, thus providing the following numbers:
```

will be affecting the load-generating connections as well as the separate latency
probing connections in the same way.

Beyond the difference in the latency of the load-generating connections and the

> separate connections another element can provide additional information. Namely

Reviewer: ```suggestion
separate connections, another element can provide additional information: Namely
```

Author: Applied.

testing against different servers located in different places along the path will

> allow to some extent to separate the network’s path in different segments. For

Reviewer: ```suggestion
allow, to some extent, the opportunity to separate the network’s path in different segments. For
```

Author: Applied.

> example, if the cable modem and a further away ISP server are hosting

Reviewer: ```suggestion
example, if the cable modem and a more-distant ISP server are hosting
```

Author: Applied, but without the hyphen. I don't think that is needed, no?

responsiveness measurement endpoints, some localization of the issue can be done.
If the RPM to the cable modem is very high, it means that the network segment
from the client endpoint to the cable modem does not have responsiveness issues,
thus allowing the user to conclude that possible responsiveness issues are
beyond the cable modem.

> It must be noted though that due to the high level approach to the testing

Reviewer: ```suggestion
It must be noted, though, that due to the high level approach to the testing
```

Both the client and the server MUST support HTTP/2 over TLS.
The client MUST be able to send a request with a GET or POST method.
The client MUST send the GET without the "Accept-Encoding" header to ensure the

> server will not compress the data.

Reviewer: I am concerned about this statement. I am reading through [12.5.3 of RFC9110](https://www.rfc-editor.org/rfc/rfc9110.html#section-12.5.3) and it says

> If no Accept-Encoding header field is in the request, any content coding is considered acceptable by the user agent. (https://www.rfc-editor.org/rfc/rfc9110.html#section-12.5.3-10.1)[Link]

Then later, it says:

> An Accept-Encoding header field with a field value that is empty implies that the user agent does not want any content coding in response. [Link](https://www.rfc-editor.org/rfc/rfc9110.html#section-12.5.3-12)

I honestly have no idea whether 9110 is the latest or not (it *seems* to be) and I also have no idea whether I am reading the information correctly. I am adding this information in case it helps.

Author: Asked our internal HTTP expert....

Reviewer: I am eager to hear what they say!

Reviewer: Thanks!! I am glad that I was able to read the standardese correctly. goresponsiveness already includes this header. Thanks for checking!
